src/app/components/elements/inputs/PasswordInput.tsx

```diff
@@ -32,6 +33,7 @@ export const PasswordInputs = ({
   setUnverifiedPassword,
   defaultPassword,
 }: PasswordInputProps) => {
+  const [PasswordInputType, ToggleIcon] = usePasswordToggle() as ["text" | "password", React.ReactNode];
```

Comment:
Just a small one. Noticed "PasswordInputType, ToggleIcon" are not camel cased compared to other consts

Reply:
That hook was removed from the code